address/address_test.go

```diff
@@ -56,3 +59,30 @@ func TestBech32(t *testing.T) {
 	}
 
 }
+
+func Test_Blech32EncodeDecode(t *testing.T) {
+	decodedWif, err := btcutil.DecodeWIF(wifTest)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	pkCompressed := decodedWif.PrivKey.PubKey().SerializeCompressed()
+	publicKeyHashCompressed := btcutil.Hash160(pkCompressed)
+
+	blech32Address := address.Bech32{}
+
+	blech32Addr, err := blech32Address.ToBlech32("el", 0x00, publicKeyHashCompressed)
+	if err != nil {
+		t.Error(err)
+	}
+
+	fromBlech32, err := blech32Address.FromBlech32(blech32Addr)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if hex.EncodeToString(fromBlech32.Data) != hex.EncodeToString(publicKeyHashCompressed) {
```

Comment:
I don't think it's enough to check just that the pubkey hash matches. 
Take a look at the [tests](https://github.com/vulpemventures/blech32/blob/master/tests/index.js) of the js version.
We should use the same test vectors.